server.js

```diff
@@ -3479,11 +3479,17 @@ cache(function(data, match, sendBadge, request) {
   var spec = match[2];  // eg, AFNetworking
   var format = match[3];
   var apiUrl = 'https://trunk.cocoapods.org/api/v1/pods/' + spec + '/specs/latest';
-  var badgeData = getBadgeData('pod', data);
+  var badgeData;
+  if (type === 'v') {
+    badgeData = getBadgeData('pod', data);
+  } else if (type === 'p') {
+    badgeData = getBadgeData('platform', data);
+  } else if (type === 'l') {
+    badgeData = getBadgeData('license', data);
+  }
   badgeData.colorscheme = null;
   request(apiUrl, function(err, res, buffer) {
-    if (err != null) {
-      badgeData.text[1] = 'inaccessible';
+    if (checkErrorResponse(badgeData, err, res, 'not found')) {
```

Comment:
Why do you pass lass argument here? Default not-found message value is `'not found'`, isn't it?